packages/sdk-server/src/sdk-client.ts

```diff
@@ -42,8 +42,8 @@ export class TurnkeyServerSDK {
     });
   };
 
-  apiProxy = async (methodName: string, params: any[]): Promise<any> => {
-    const apiClient = this.api();
+  apiProxyClient = async (methodName: string, params: any[]): Promise<any> => {
```

Comment:
@AlwaysBCoding lmk if you want to leave this as `apiProxy` instead

Reply:
I think we should leave it as `apiProxy` for now because it's not really an http client the same way the apiClient, passkeyClient, and iframeClient are